doc/getting-started/getting-started/installation.md

```diff
@@ -134,6 +134,7 @@ To build cln to just install a tagged or master version you can use the followin
 pip3 install --upgrade pip
 pip3 install mako
 pip3 install -r plugins/clnrest/requirements.txt
+pip3 install grpcio-tools
```

Comment:
mh this is strange, it is required to install core lightning? `grpcio-tools` should be used only in tests

Reply:
Without this, the build fails.

Reply:
Mh this is nice to know :/

Maybe clnrest -> pyln-client -> uses the girpcio packages and maybe the `clnrest/requirements.txt` is not completed.

idk but this required more exploration

Reply:
It should only need `grpcio-tools` when the `.proto` files have a newer timestamp than the corresponding `_pb2.py` files. Might be a missing generation update?

Reply:
This is using the release tag? If that fails requiring grpcio-tools then it is likely correct to keep those instructions in there.